src/unix/udp.c

```diff
@@ -1298,7 +1298,8 @@ static int uv__udp_sendmsgv(int fd,
   r = 0;
   nsent = 0;
 
-#if defined(__linux__) || defined(__FreeBSD__) || defined(__APPLE__)
+#if defined(__linux__) || defined(__FreeBSD__) || defined(__APPLE__) || \
```

Comment:
There is a comment on line 1329 in the corresponding `#endif`. Can you update that too please.

Reply:
Sure, I added the missing test without the continuation character.  Thanks.